utils/strings/strings.go

```diff
@@ -158,28 +158,30 @@ func Merge(ms ...[]string) []string {
 	return base
 }
 
-// ConvertToMap :if env list containers Unicode punctuation character,will ignore this element.
-func ConvertToMap(env []string) map[string]string {
-	envs := map[string]string{}
-	var k, v string
-	for _, e := range env {
-		if e == "" {
+// ConvertEnv Convert []string to map[string]interface{}
+func ConvertEnv(envList []string) (env map[string]interface{}) {
```

Comment:
In my option, as common utils with package string, it's weird to see the word env here.
Would a generic field like string `stringSlice`, `stringMap`,`stringInterfaceMap`, and so on be more appropriate?